packages/web/src/scss/components/Grid/README.md

```diff
@@ -114,7 +114,49 @@ Custom vertical (y-axis) spacing from tablet up:
 </ul>
 ```
 
-## Grid Item
+## Item Alignment
+
+The `Grid--alignmentX*` and `Grid--alignmentY*` classes are used to control the alignment of items within a container.
+The available values for these classes can be found in our [alignment dictionary][alignment-dictionary].
+
+`Grid--alignmentX*`: Manages horizontal alignment, allowing you to position items to the left, center, or right of the container. It can also be configured with responsive values for different breakpoints.
+`Grid--alignmentY*`: Manages vertical alignment, enabling you to position items at the top, center, or bottom of the container. It supports responsive values for various breakpoints as well.
+
+Horizontal Alignment:
+
+```html
+<Grid class="Grid Grid--alignmentXLeft">
```

Comment:
Since an alignment will always be provided by the React and Twig implementation of `Grid`, these class names will always be there:

- `.Grid--alignmentXStretch`
- `.Grid--alignmentYStretch`

Please, would you adjust all examples in the web README and web docs so the alignmentX/Y class names are always present?

It means that examples like this:

```html
<div class="Grid"></div>
```

would become:

```html
<div class="Grid Grid--alignmentXStretch Grid--alignmentYStretch"></div>
```

cc @crishpeen

Reply:
What do you mean by saying that stretch classes will always be there? The default is 'normal,' which has no class. The stretch class should only be present when you set the alignment to stretch.

Reply:
We discussed this with Adam and we suggest to copy the behaviour from other components, eg. Button, where we add "default" classes to the DOM (in Button the size and colour classes, standalone `.Button` has no colour or size). So there will always be some `.Grid--alignmentX*` and some `.Grid--alignmentY*` classes in all technologies. This way we can make the `stretch` really the default. 

So we "don't" support no alignment class. In React and Twig we will add the default automatically, in the web it will always be there (but of course users can avoid it, but we don't support it).

Reply:
Default changed to stretch and classes in react and twig are present in this default state